src/modules/chat_left_side/index.js

```diff
@@ -1,18 +1,21 @@
 import {ChatLayoutTypes, PlatformTypes, SettingIds} from '../../constants.js';
+import dom from '../../observers/dom.js';
 import settings from '../../settings.js';
 import {loadModuleForPlatforms} from '../../utils/modules.js';
 
+const SIDE_NAV_SELECTOR = '.side-nav';
+
 class ChatLeftSide {
   constructor() {
     settings.on(`changed.${SettingIds.CHAT_LAYOUT}`, () => this.toggleLeftSideChat());
+    dom.on(SIDE_NAV_SELECTOR, () => this.toggleLeftSideChat());
     this.toggleLeftSideChat();
   }
 
   toggleLeftSideChat() {
     document.body.classList.toggle('bttv-swap-chat', settings.get(SettingIds.CHAT_LAYOUT) === ChatLayoutTypes.LEFT);
 
-    // TODO: replace this eventually when :has exists in CSS
```

Comment:
https://caniuse.com/css-has is it time?

Reply:
`:has` will immediately grab the highest element in the tree that meets this condition. so the root is returned

What we want is `div > .side-nav` I believe